src/firefly/js/metaConvert/AtlasRequestList.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+
+import {get} from 'lodash';
+import {makeServerRequestBuilder} from './converterUtils.js';
+import {RangeValues,STRETCH_LINEAR,PERCENTAGE} from '../visualize/RangeValues.js';
+import {getCellValue} from '../tables/TableUtil.js';
+import {parseWorldPt} from '../visualize/Point.js';
+import {convertAngle} from '../visualize/VisUtil.js';
+
+const colToUse= [ 'facility_name', 'band_name', 'instrument_name', 'fname', 'wavelength',
+    'file_type', 'in_ra', 'in_dec', 'dataproduct_type'];
+const rangeValues= RangeValues.makeRV({which:PERCENTAGE, lowerValue:1, upperValue:99, algorithm:STRETCH_LINEAR});
+
+const bandIdx= {'IRAC1':0, 'IRAC2':1, 'MIPS24':2};
+
+/**
+ * make a list of plot request for atlas. This function works with ConverterFactory.
+ * @param table
+ * @param row
+ * @param includeSingle
+ * @param includeStandard
+ * @param threeColorOps
+ * @return {{}}
+ */
+export function makeAtlasPlotRequest(table, row, includeSingle, includeStandard, threeColorOps) {
+
+    var headerParams= ['mission', 'ds', 'band','dataset', 'table'];
+
+    const svcBuilder= makeServerRequestBuilder(table,colToUse,headerParams,rangeValues,1);
+    const builder = (plotId, reqKey, title, rowNum, extraParams) => {
+        const req= svcBuilder(plotId, reqKey, title, rowNum, extraParams);
+        const subsize = get(table, 'request.subsize');
+        if (subsize && subsize>0) {
+            const {UserTargetWorldPt, sizeUnit} = get(table, 'request', {});
+            if (UserTargetWorldPt && sizeUnit) {
+                const wp = parseWorldPt(UserTargetWorldPt);
+                // cutout is requested when in_ra, in_dec, and subsize are set (see AtlasIbeDataSource)
+                req.setParam('center', `${wp.getLon()},${wp.getLat()}`); // degrees assumed if no unit
+                req.setParam('in_ra', `${wp.getLon()}`);
+                req.setParam('in_dec', `${wp.getLat()}`);
```

Comment:
This appears to work but something does not make sense?  The `AtlasIbeDataSource` want to user `UserTargetWorldPt` but you are passing `in_ra` and `in_dec`.  I think you need to call `req.setWorldPt`

Also I think you want to build the WebPlotRequest based on the row not no the original table search request.

Reply:
yes, in_ra, in_dec is from the row. I took this code directly from the wiseRequestList and i'm sure is not complete. See below. Sorry.